### ISSUE TITLE
Documentation contains incorrect instructions to reference hosted checkout /dist auto-loader in Checkout (Advanced Options)

### ISSUE THREAD

**Issue by Mikhail-MM** [bigcommerce/checkout-js#475]
Excited to work with the Optimized One Page Checkout source code finally - but had some trouble getting the package built & hosted correctly.

To test that my build/hosting process is correct, I made a few changes to `Checkout.tsx` - just a few custom console.logs and classNames to the React components as a sanity check to make sure that my changes were being applied.

I'm following the instructions from the BigCommerce Documentation:
https://developer.bigcommerce.com/stencil-docs/customizing-checkout/installing-custom-checkouts
```
To install a custom checkout on a store, follow these steps:

Navigate to Advanced Settings > Checkout in your store’s control panel.

Under Checkout Type, select Custom Checkouts.

Type the following in the Script URL field, replacing with the latest version number generated by running the npm run release command:

**webdav:checkout/auto-loader-<version>.js**
```

First thing to mention is that using `npm release` in this repo as the docs suggest does not work, and returns this error:
```
>echo 'Please do not relase locally, use CircleCi'
```

So, after looking at the dist folder, there was a file named `auto-loader-1.104.1`, so I used the following URL:

```
webdav:checkout/auto-loader-1.1014.1.js
```

When I logged into my live store, I saw the checkout, however my custom changes were missing completely. Now, it **was** still pulling _something_ from WebDAV because when I changed the version number to something that did not exist, i.e. `auto-loader-12345.js`, the checkout page was blank and failed to load completely.

So, how I solved this was noticing that there is a file called `auto-loader.js` without any version #.

Using the following URL, my changes were able to load successfully:

```
webdav:checkout/auto-loader.js
```

So really, no version number is required when pointing to the auto-loader in the BigCommerce Control Panel, and that's quite confusing. 

In addition, you guys may want to mention something about the build process not working at all on Windows, this may save people some headaches. I was running into cryptic npm errors and half-built dist/build folders until I switched to my MacOS laptop to give it a try.

**Comment by capsula4** [comment]
Hi Mikhail, thanks for raising this issue. 

While we look into it, we advise that you run instead `npm run release:alpha` to test your changes.

The version number is not required, but highly recommended. If you make changes and you don't use the new version number, a cached version could be served to the users.

**Comment by capsula4** [comment]
https://github.com/bigcommerce/checkout-js/pull/476

**Comment by Mikhail-MM** [comment]
@capsula4 As an additional note, I've tried to run this in WSL (Linux SubSystem for Windows) - I couldn't get things running. Could be different with WSL2. Not sure.

**Comment by Mikhail-MM** [comment]
@capsula4 Running the release with `npm run release:alpha` seemed to auto-prefix a new version number each time (based on time-stamp, looks like).

I was able to successfully get the custom checkout rendering after uploading it to WebDAV. 

Got an error with eslint not finding the airbnb config, might be a weird quirk, so I axed that from the script, and afterwards everything seemed to work. I will try to look into why this failed for me earlier, I might have missed something.

**Comment by Mikhail-MM** [comment]
@capsula4  Just one question - there is no way to get this running locally inside of cornerstone, without running the build and hosting the files externally as a separate package?

Your team had previously released a smaller, self-contained React Checkout example implementation to demo the checkout-sdk, and there were steps laid out that allowed us to embed the React source assets directly inside of cornerstone.

References to this method seemed to have been wiped but have been preserved here:
https://github.com/bigcommerce/dev-docs/blob/3d5a630524f25e935789da7f72404be386659284/docs/stencil-docs/customizing-checkout/checkout-sdk-example.md

Allowing us to keep the checkout assets inside of the cornerstone theme was convenient in several ways - we could version out releases from staging to production quickly, and rolling back was instantaneous and allowed us to utilize BigCommerce's theming version control. 

With the current set-up, I don't see how we would be able to serve one version of the checkout to customers on a live store while developing a version for staging simultaneously, development of a staging theme would need to be done on a separate sandbox store, is that correct?

None of the instructions include any information on running the typescript checkout package separately on your host machine, stand-alone - I have tried to run it locally in development but it seems like it's just an http server that maps to the `/build` directory containing a separate, development-only build of compiled files. I assume they need the SSR injection that BigCommerce provides for the checkout to actually render? I navigated to localhost:8080 but I couldn't really get this running on my machine without navigating to the actual live BigCommerce domain.

I feel like theoretically it would be possible to incorporate this into cornerstone and run both within stencil-cli, but the webpack configuration for this project is pretty involved, I think it would be difficult to reconcile the dependencies of both packages.
